esignet-mock.properties

```diff
@@ -419,7 +419,8 @@ mosip.esignet.vci.key-values={ 'credential_issuer': '${mosip.esignet.vci.identif
   'gender': {'display': {{'name': 'Gender','locale': 'en'}}},\
   'benefits': {'display': {{'name': 'Benefits','locale': 'en'}}},\
   'email': {'display': {{'name': 'Email Id','locale': 'en'}}},\
-  'expiresOn': {'display': {{'name': 'Expiry','locale': 'en'}}},\
+  'policyIssuedOn': {'display': {{'name': 'Expiry','locale': 'en'}}},\
+  'policyExpiresOn': {'display': {{'name': 'Expiry','locale': 'en'}}},\
```

Comment:
Same to be taken care here https://github.com/mosip/mosip-config/pull/5303#discussion_r1483073571

Reply:
changed